src/tests/functional/inference_engine/lp_transformations/disable_convert_on_const_path_transformation.cpp

```diff
@@ -132,7 +132,7 @@ const std::vector<DisableConvertOnConstPathTransformationValues> testValues = {
             },
             {
                 { ngraph::element::f32, false },
-                { {128.f}, element::f32, {}, false, 1ul, element::i8, true },
+                { {128.f}, element::f32, {}, false, 1ul, element::u8, true },
```

Comment:
Can we change the constant value instead of precision to be aligned with the comment above?
```suggestion
                { {127.f}, element::f32, {}, false, 1ul, element::i8, true },
```